addons/mail/static/src/components/message/message.js

```diff
@@ -649,6 +650,18 @@ class Message extends Component {
         }
     }
 
+    /**
+     * @private
+     * @param {MouseEvent} ev
+     */
+    _onClickUnfollow(ev) {
+        if (!this.message.originThread) {
+          return;
+        }
+        this.message.markAsRead();
+        this.message.originThread.unfollow();
```

Comment:
Functionally question here: do we actually want to completely unfollow? Or just unfollow from the subtype of the message?